scripts/date-snapshot/date-snapshot.sh

```diff
@@ -49,14 +79,14 @@ while test $# -gt 0; do
       break
       ;;
     -f|--file)
-	  shift
+      shift
       while read line; do
-      download_checkout $line		
+        download_checkout $line
       done <$1
-	  break
+      break
       ;;
     *)
-      echo "Usage: ${package} [-s] [-i|--input] [-f|--file]"
+      echo "Usage: ${package} [-c|--clean] [-d|delete] [-i|--input] [-f|--file] <input>"
```

Comment:
adding -h/--help and have it spit out your help text is pretty normal too (this is a nicety not required)